### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by davidhao3300**
screen shot 2015-05-01 at 2 45 13 pm

### CODE REVIEW

**Comment by davidhao3300** [comment]
The bottom has a border, the calendar was too large to show all of it.

File: cal/static/css/style.css [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+}
+
+.event:nth-child(odd) > a {
+}
+
+.event:nth-child(even) {
+}
+
+.event:nth-child(even) > a {
 }
```
Comment (alanhdu) [inline cal/static/css/style.css]: What are these for?

File: cal/static/jsx/app.jsx [hunk 2]
```diff
@@ -59,6 +59,11 @@ var RApp = React.createClass({
     render: function() {
         return (
             <div className="app">
+                <RSearch setGlobalState={this.setState.bind(this)}
```
Comment (alanhdu) [inline cal/static/jsx/app.jsx]: Do we need to `bind(this)?`. I think we can get away with just `this.setState`.

File: cal/static/jsx/calendar.jsx [hunk 3]
```diff
@@ -30,14 +33,7 @@ var REvent = React.createClass({
...
 
-            // Only add start time's AM/PM if start and end dates are
-            // different, or start and end are in different halves of
-            // the day.
-            equal_dates = start.toDateString() == end.toDateString();
-            if (equal_dates || AMPM(start) != AMPM(end)) {
-                timeString += AMPM(start);
-            }
-            timeString += " \u2013 " + formatTime(end) + AMPM(end);
+            timeString += AMPM(start) + " \u2013 " + formatTime(end) + AMPM(end);
```
Comment (alanhdu) [inline cal/static/jsx/calendar.jsx:37]: Could we comment what `\u2013` is? It's a dash, right (&#8211;)?

**Pull request merged**